index.html

```diff
@@ -113,6 +113,31 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Uh oh, looks like this was place in the wrong place.

It's not your fault, there was another card in the wrong spot near by too.

You actually want to put this in [just below here](https://github.com/Syknapse/Contribute-To-This-Project/pull/3668/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L147-L151). Can you please move it down? 

Here's a screenshot to make it easier to see where I'm talking about:

![image](https://github.com/Syknapse/Contribute-To-This-Project/assets/992688/dea67f81-d7cf-4bb7-a46c-9f3b2a79c240)

Sorry about that, but other than that the card itself looks great! 🎉

Reply:
corrected, thank you.